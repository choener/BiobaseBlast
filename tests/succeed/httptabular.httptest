# blastn
# Iteration: 0
# Query: Aeromonas salmonicida subsp. salmonicida A449
# RID: SK8PG80901R
# Database: nt
# Fields: query id, subject ids, query acc.ver, subject acc.ver, % identity, alignment length, mismatches, gap opens, q. start, q. end, s. start, s. end, evalue, bit score
# 9 hits found
Aeromonas	gi|1214238400|gb|CP022186.1|	Aeromonas	CP022186.1	100.000	62	0	0	1	62	4466360	4466299	3.85e-22	113
Aeromonas	gi|1214232668|gb|CP022181.1|	Aeromonas	CP022181.1	100.000	62	0	0	1	62	4451525	4451464	3.85e-22	113
Aeromonas	gi|1214226508|gb|CP022175.1|	Aeromonas	CP022175.1	100.000	62	0	0	1	62	4459524	4459463	3.85e-22	113
Aeromonas	gi|1202187378|gb|CP021654.1|	Aeromonas	CP021654.1	100.000	62	0	0	1	62	4130312	4130373	3.85e-22	113
Aeromonas	gi|142849896|gb|CP000644.1|	Aeromonas	CP000644.1	100.000	62	0	0	1	62	197460	197521	3.85e-22	113
Aeromonas	gi|1024308492|gb|CP015448.1|	Aeromonas	CP015448.1	95.161	62	2	1	1	62	690724	690664	1.03e-16	95.1
Aeromonas	gi|1024268833|gb|CP014774.1|	Aeromonas	CP014774.1	95.161	62	2	1	1	62	2959093	2959153	1.03e-16	95.1
Aeromonas	gi|1006067808|gb|CP012504.1|	Aeromonas	CP012504.1	95.161	62	2	1	1	62	391011	391071	1.03e-16	95.1
Aeromonas	gi|328802836|gb|CP002607.1|	Aeromonas	CP002607.1	95.161	62	2	1	1	62	167317	167377	1.03e-16	95.1




