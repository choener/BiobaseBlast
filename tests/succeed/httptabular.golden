BlastTabularResult {blastProgram = BlastN, blastQueryId = "Aeromonas", blastDatabase = "nt", blastHitNumber = 9, hitLines = [BlastTabularHit {queryId = "Aeromonas", subjectId = "gi|1214238400|gb|CP022186.1|", seqIdentity = 100.0, alignmentLength = 62, misMatches = 0, gapOpenScore = 0, queryStart = 1, queryEnd = 62, hitSeqStart = 4466360, hitSeqEnd = 4466299, eValue = 3.85e-22, bitScore = 113.0, subjectFrame = 0, querySeq = "", subjectSeq = ""},BlastTabularHit {queryId = "Aeromonas", subjectId = "gi|1214232668|gb|CP022181.1|", seqIdentity = 100.0, alignmentLength = 62, misMatches = 0, gapOpenScore = 0, queryStart = 1, queryEnd = 62, hitSeqStart = 4451525, hitSeqEnd = 4451464, eValue = 3.85e-22, bitScore = 113.0, subjectFrame = 0, querySeq = "", subjectSeq = ""},BlastTabularHit {queryId = "Aeromonas", subjectId = "gi|1214226508|gb|CP022175.1|", seqIdentity = 100.0, alignmentLength = 62, misMatches = 0, gapOpenScore = 0, queryStart = 1, queryEnd = 62, hitSeqStart = 4459524, hitSeqEnd = 4459463, eValue = 3.85e-22, bitScore = 113.0, subjectFrame = 0, querySeq = "", subjectSeq = ""},BlastTabularHit {queryId = "Aeromonas", subjectId = "gi|1202187378|gb|CP021654.1|", seqIdentity = 100.0, alignmentLength = 62, misMatches = 0, gapOpenScore = 0, queryStart = 1, queryEnd = 62, hitSeqStart = 4130312, hitSeqEnd = 4130373, eValue = 3.85e-22, bitScore = 113.0, subjectFrame = 0, querySeq = "", subjectSeq = ""},BlastTabularHit {queryId = "Aeromonas", subjectId = "gi|142849896|gb|CP000644.1|", seqIdentity = 100.0, alignmentLength = 62, misMatches = 0, gapOpenScore = 0, queryStart = 1, queryEnd = 62, hitSeqStart = 197460, hitSeqEnd = 197521, eValue = 3.85e-22, bitScore = 113.0, subjectFrame = 0, querySeq = "", subjectSeq = ""},BlastTabularHit {queryId = "Aeromonas", subjectId = "gi|1024308492|gb|CP015448.1|", seqIdentity = 95.161, alignmentLength = 62, misMatches = 2, gapOpenScore = 1, queryStart = 1, queryEnd = 62, hitSeqStart = 690724, hitSeqEnd = 690664, eValue = 1.03e-16, bitScore = 95.1, subjectFrame = 0, querySeq = "", subjectSeq = ""},BlastTabularHit {queryId = "Aeromonas", subjectId = "gi|1024268833|gb|CP014774.1|", seqIdentity = 95.161, alignmentLength = 62, misMatches = 2, gapOpenScore = 1, queryStart = 1, queryEnd = 62, hitSeqStart = 2959093, hitSeqEnd = 2959153, eValue = 1.03e-16, bitScore = 95.1, subjectFrame = 0, querySeq = "", subjectSeq = ""},BlastTabularHit {queryId = "Aeromonas", subjectId = "gi|1006067808|gb|CP012504.1|", seqIdentity = 95.161, alignmentLength = 62, misMatches = 2, gapOpenScore = 1, queryStart = 1, queryEnd = 62, hitSeqStart = 391011, hitSeqEnd = 391071, eValue = 1.03e-16, bitScore = 95.1, subjectFrame = 0, querySeq = "", subjectSeq = ""},BlastTabularHit {queryId = "Aeromonas", subjectId = "gi|328802836|gb|CP002607.1|", seqIdentity = 95.161, alignmentLength = 62, misMatches = 2, gapOpenScore = 1, queryStart = 1, queryEnd = 62, hitSeqStart = 167317, hitSeqEnd = 167377, eValue = 1.03e-16, bitScore = 95.1, subjectFrame = 0, querySeq = "", subjectSeq = ""}]}
